src/client/components/Notification.jsx

```diff
@@ -0,0 +1,11 @@
+export default function Notification({ message, setNotification }) {
```

Comment:
This is cool. I did lots of state values as notifications which is far less effective (wanted to refactor over the weekend but had no time in the end)